src/qt/forms/optionsdialog.ui

```diff
@@ -33,7 +33,7 @@
           <string>Automatically start %1 after logging in to the system.</string>
          </property>
          <property name="text">
-          <string>&amp;Start %1 on system login</string>
```

Comment:
Not sure it's a good idea to tie the shortcut to PACKAGE_NAME... This option is more about startup anyway: [S] is a good fit.